app/views/languages/repositories.erb

```diff
@@ -0,0 +1,58 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Exercism Repositories</h1>
+    <p class="lead">
+      <strong>What are the repositories you can contribute to?</strong>
+      <br />
+    </p>
+    <p> Exercism is a project comprised of several repositories and these can be divided basically into:
+    </p>
+    <h2> Basic components </h2>
+    <ul>
+      <li><strong><a href="https://github.com/exercism/exercism.io">The website</a></strong>: Holds the code that people submit. Browse and review solutions, and have conversations.</li>
+      <li><strong><a href="https://github.com/exercism/cli">Command line interface</a></strong>: Use to fetch exercises and submit solutions.</li>
+      <li><strong><a href="https://github.com/exercism/x-api">Problems API</a></strong>: Delivers exercises. It is used by the Exercism command-line client, as well as the Exercism website.</li>
+    </ul>
+    <h2> Exercises </h2>
+    <ul>
+    <li><strong><a href="https://github.com/exercism/x-common">X-Common repository</a></strong>: Shares language-independent problem descriptions for Exercism exercises.</li>
```

Comment:
We also need to refer to all the language-specific implementations of the exercises, which are listed below.